include/gazebo_vision_plugin.h

```diff
@@ -56,6 +56,15 @@
 
 #include <Odometry.pb.h>
 
+// IF ROS DEFINED
```

Comment:
Use [`add_definitions()`](https://cmake.org/cmake/help/v3.0/command/add_definitions.html) on the `CMakeLists.txt`:
```make
add_definitions(-DBUILD_ROS_INTERFACE=${BUILD_ROS_INTERFACE})
```
Then you can use them on code:
```c++
#if BUILD_ROS_INTERFACE == ON // or 1, not sure
// ROS Topic subscriber
#include <thread>
#include <ros/ros.h>
...
#endif
```

Reply:
Thanks, it works.